private-templates-service/adaptivecards-templating-service/src/tests/router.test.ts

```diff
@@ -9,32 +9,33 @@ import { InMemoryDBProvider } from "../storageproviders/InMemoryDBProvider";
 import { ITemplate, ITemplateInstance } from "../models/models";
 
 export default async function getToken(): Promise<string> {
-  const request = require("request-promise");
-  const endpoint = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/oauth2/token";
-  const requestParams = {
-    grant_type: "client_credentials",
-    client_id: "#{CLIENT_ID_TOKEN}#",
-    client_secret: "#{CLIENT_SECRET_TOKEN}#",
-    resource: "#{CLIENT_ID_TOKEN}#"
-  };
```

Comment:
is this for testing? 
I would not recommend checking in commented code

Reply:
My mistake that was accidentally added, fixed!